pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumer.java

```diff
@@ -159,6 +163,16 @@ private PulsarKafkaConsumer(ConsumerConfig config, Deserializer<K> keyDeserializ
         }
     }
 
+    private OffsetResetStrategy getStrategy(final String strategy) {
+    	if (strategy.equals("earliest")) {
```

Comment:
couldn't `strategy` be null?

Reply:
Yeah, will change to the logic to take that into account.